pkg/loadbalancer/roundrobin.go

```diff
@@ -142,9 +142,10 @@ func (rr *roundRobin) getTargetAndIncrement(filteredNodes []*api.Node, selfNodeI
 		targetNodeNumber int
 		isRemoteConn     bool
 	)
-	if rr.nextCreateNodeNumber != 0 {
-		targetNodeNumber = rr.nextCreateNodeNumber
+	if len(filteredNodes) == 0 {
+		return "", false
```

Comment:
I think `len(filterNodes)` will not be 0. However, since we are checking for that, how will callers react to the empty string as the first return parameter? Will they be able to handle empty mgmt IP?

Reply:
Yeah, i agree, it won't be zero.
I just wanna make sure whenever this codebase is expanded, we won't run into null pointer or index out of bound. 

Current caller checks the node length before passing it to the function.